addons/sale/tests/test_sale_order.py

```diff
@@ -639,6 +639,48 @@ def test_sale_order_line_product_taxes_on_branch(self):
             {'product_id': product_no_tax.id, 'tax_id': []},
         ])
 
+    def test_price_recomputation_on_readonly_unit_price(self):
+        """Make sure that price computation works fine when unit price is readonly.
+
+        Since the client doesn't send readonly fields, flagging the field as readonly
+        will result in the 'price_unit' being absent from the values, but not the
+        'technical_price_unit' field, which would disable the price computation.
```

Comment:
Nit (for consistency with the next paragraph):
```suggestion
        will result in the `price_unit` being absent from the values, but not the
        `technical_price_unit` field, which would disable the price computation.
```